src/frontend/src/lib/flows/authorize/index.json

```diff
@@ -7,18 +7,15 @@
     "waiting_for_auth_data": "Hang tight, connecting to the dapp",
     "validating_auth_data": "Hang tight, validating the dapp's information",
     "finalizing_auth": "Hang tight, starting your session",
-
     "invalid_request": "Invalid request",
     "invalid_authentication_request_received": "It seems like an invalid authentication request was received.",
     "connection_closed": "Connection closed",
     "connection_could_not_be_established": "It seems like the connection with the service could not be established.",
-    "wrong_place": "Wrong place",
-    "no_request_received": "Seems you arrived here for authentication, but no service has requested it.",
+    "missing_request": "Missing request",
+    "no_request_received": "We couldn't process your authentication request. Try a different browser; if the issue persists, contact the developer.",
```

Comment:
Should we add that the "The browser blocked the connection with the application. Try a different browser; if the issue persists, contact the developer." or more generic "There was an issue connecting with the application. Try a different browser..."?